README.md

```diff
@@ -61,6 +61,17 @@ To build the documentation locally, run `yarn install && yarn build && yarn docs
 in this repository. This will generate a `./docs` directory in each package that you
 can browse locally to see API docs on the various packages.
 
+## FAQ For Potential Issues
```

Comment:
Thank you for the FAQ.
This will be very useful to jog our memory in a few months (or weeks)